.drone.yml

```diff
@@ -39,7 +39,7 @@ name: push-build-linux-amd64
 environment:
   BUILDBOX_VERSION: teleport14
   GID: "1000"
-  RUNTIME: go1.20.4
+  RUNTIME: go1.20.5
```

Comment:
Are we not concerned by https://groups.google.com/g/golang-announce/c/q5135a9d924 ?

Reply:
I don't think we use `gccgo` or GoLLVM. Unless I'm mistaken we should be fine.

Reply:
Go 1.20.5 "works for me", so we should be fine.

Reply:
I haven't had problems with the libfido2 or touchid stuff, so assuming it's fine too. If have any problems, just rollback. ;)